python/ql/test/library-tests/PointsTo/new/PointsToMissing.ql

```diff
@@ -0,0 +1,26 @@
+import python
+import Util
+import semmle.python.pointsto.PointsTo
+import semmle.python.objects.ObjectInternal
+
+/* Ideally, this test should return _no_ results. */
```

Comment:
```suggestion
/* This test should return _no_ results. */
```

When I read the _Ideally_ part, I assumed that it actually would return results :confused:

Reply:
Done!